Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java

```diff
@@ -198,6 +199,11 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
 
         NotesActivity notesActivity = (NotesActivity) getActivity();
 
+        if (ACTION_NEW_NOTE.equals(notesActivity.getIntent().getAction())){
+            //if user tap on "app shortcut", create a new note
```

Comment:
We should bail out here if the user isn't signed in so a new note isn't created in that case. You can use `NotesActivity.userIsUnauthorized()` for that. It's ok to make that private method public, too.

Reply:
Should we show an error message or exit the app if the user is unauthorized or both?

Reply:
Never mind, it actually redirects to LoginActivity